plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/command/RemoveConnectionHandler.java

```diff
@@ -31,6 +31,9 @@
 	@Override
 	public Object execute(ExecutionEvent event) throws ExecutionException {
 		IConnection[] connections = UIUtils.getElements(HandlerUtil.getCurrentSelection(event), IConnection.class);
+		if( connections.length == 0 )
+			return null;
```

Comment:
Null connections are now filtered out by UIUtils, but it's still possible (somehow) that something's slipped through, so I just add that as a safety. But it's extremely unlikely it'd ever be hit.